pkg/crc/cluster/monitoring.go

```diff
@@ -0,0 +1,37 @@
+package cluster
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/code-ready/crc/pkg/crc/oc"
+	v1 "github.com/openshift/api/config/v1"
+)
+
+func StartMonitoring(ocConfig oc.Config) error {
+	data, _, err := ocConfig.RunOcCommand("get", "clusterversion/version", "-o", "json")
+	if err != nil {
+		return err
+	}
+
+	var cv v1.ClusterVersion
+	if err := json.Unmarshal([]byte(data), &cv); err != nil {
+		return err
+	}
+
+	pos := -1
+	for i, override := range cv.Spec.Overrides {
+		if override.Name == "cluster-monitoring-operator" {
+			pos = i
```

Comment:
shouldn't we use `break` as soon as we find this operator and the position?

Reply:
👍